client/reader/stream/x-post.jsx

```diff
@@ -80,9 +80,10 @@ class CrossPost extends PureComponent {
 
 	handleClick = ( event ) => {
 		if ( event.button > 0 || event.metaKey || event.controlKey || event.shiftKey || event.altKey ) {
-			return;
+			window.open( this.props.post.URL, '_blank', 'noopener noreferrer' );
```

Comment:
Can we do this the other way around, keeping the `href` and the standard link behaviour, and using `event.preventDefault()` when we don't want it?